keyboards/fluorite/config.h

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright 2019 REPLACE_WITH_YOUR_NAME
```

Comment:
Please fix these.

Reply:
Copyright 2019 Mafuyu Ihotsuno
I was wondering if you could replace my name, please.

Reply:
I don't have permissions to commit to your repo, you must do this yourself. Just commit to the same branch and it will be added to the PR.

Reply:
I am sorry. I understand. thanks!